locales/en-US/messages.properties

```diff
@@ -68,6 +68,86 @@ navViewTutorial=TUTORIAL
 navToggleAutoUpdateTitle=Toggle Preview Auto-refresh
 navToggleAutoUpdateLabel=Auto
 
+# Code Snippet Data
+snippetHTMLComment=Comment
+snippetHTMLCommentTitle=Add a Comment
+snippetHTMLCommentData=<!-- Your HTML comment here -->\n
```

Comment:
Thanks for asking, @gideonthomas :) That’s indeed tricky, and I’m not sure there’s a perfect solution. Having a loooong string with lots of `\n` and embedded code would work, technically, but I think what would be better here is extracting the text from the code snippets, then doing the formatting on the code side.

To be more explicit:
`snippetHTMLCommentData=Your HTML comment here`

In .js file, somewhere:
`var myString = '<!--' + l10nLib.getString('snippetHTMLCommentData') + '-->\n';`

Concatenating strings is generally a bad practice, but in this case it’s actually fine since we don’t want localizers to change the markup/code. Would that work with the rest of the snippets? It can be okay to include pieces of code when it makes string extraction easier, as long as there is a comment saying that this part should not be translated.

In the example above, we would add a comment explaining the context (concatenated with `<!-- -->`)

Does that make sense, or maybe I’m missing something?